src/plugins/embeddable/public/lib/embeddables/error_embeddable.tsx

```diff
@@ -11,6 +11,7 @@ import { EmbeddablePanelError } from '../panel/embeddable_panel_error';
 import { Embeddable } from './embeddable';
 import { EmbeddableInput, EmbeddableOutput, IEmbeddable } from './i_embeddable';
 import { IContainer } from '../containers';
+import './error_embeddable.scss';
```

Comment:
The old `error_embedabble.scss` wasn't actually being used anywhere - now it is 👍